Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```

Comment:
needed?

Reply:
Not needed, just standard bundler gem generation garbage. I'll remove.